### PR TITLE
feat: new wallet bar for plugin

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes 

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/shared/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "@masknet/theme": "workspace:*",
     "@masknet/web3-shared-base": "workspace:*",
     "@masknet/web3-shared-evm": "workspace:*",
+    "@masknet/plugin-wallet": "workspace:*",
```
Comment (Jack-Works) [inline packages/shared/package.json]: Is it OK? No circular reference?

File: packages/shared/src/UI/components/WalletStatusBar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        color: MaskColorVar.twitterButtonText,
+        position: 'absolute',
+        top: theme.spacing(1),
+        left: `calc(50%-${theme.spacing(1)})`,
+    },
+}))
+interface WalletStatusBarProps extends withClasses<'button'> {
+    iconSize?: number
+    badgeSize?: number
+    actionProps?: {
```
Comment (Jack-Works) [inline packages/shared/src/UI/components/WalletStatusBar/index.tsx:47]: Flatten this prop?

**Review by septs — CHANGES_REQUESTED** [review]
plz use https://aka.pw/svgomg reduce svg image

File: packages/shared/tsconfig.json [hunk 3]
```diff
@@ -13,6 +13,7 @@
     { "path": "../shared-base" },
     { "path": "../shared-base-ui" },
     { "path": "../icons" },
-    { "path": "../plugin-infra" }
+    { "path": "../plugin-infra" },
+    { "path": "../plugins/Wallet/" }
```
Comment (Jack-Works) [inline packages/shared/tsconfig.json]: Does it create a circular reference? 🤔

File: packages/shared/package.json (hunk 1 above)
Comment (lelenei) [reply packages/shared/package.json]: it is OK!

File: packages/shared/src/UI/components/WalletStatusBar/index.tsx (hunk 2 above)
Comment (lelenei) [reply packages/shared/src/UI/components/WalletStatusBar/index.tsx:47]: image

**Comment by lelenei** [comment]
remove

**Pull request closed without merging**